.daily.yml

```diff
@@ -1,3 +1,8 @@
+pr:
+  branches:
+    include:
+      - master
```

Comment:
This pipeline is currently running for every PR. I think we want `pr: none`.

https://docs.microsoft.com/en-us/azure/devops/pipelines/build/triggers?view=azure-devops&tabs=yaml#pr-triggers

Reply:
Yes I know, I've put that back on purpose so I can make changes on this PR and see it run.

Reply:
(I did try to add a branch filter, but I couldn't get that to work because it comes from a fork).